### PR TITLE
[DOCS] 7.17.4 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/1892.

Previews:
- [Index page](https://security-docs_1898.docs-preview.app.elstc.co/guide/en/security/7.17/release-notes.html)
- [Release notes](https://security-docs_1898.docs-preview.app.elstc.co/guide/en/security/7.17/release-notes-header-7.17.0.html#release-notes-7.17.4)

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
@peteharverson let me know that https://github.com/elastic/kibana/pull/130372 needed to be release-noted for 7.17.4, so I've added it to the draft. Will also need to make a few doc updates to the Security and Kibana docs to note this change. Have filed https://github.com/elastic/security-docs/issues/1903 to track that work. 

cc: @cnasikas @academo

**Review by joepeeples — APPROVED** [review]

**Review by cnasikas — APPROVED** [review]

**Comment by cnasikas** [comment]
@nastasha-solomon Should we do the same for the Kibana release notes?

**Review by benironside — APPROVED** [review]

File: docs/release-notes/7.17.asciidoc [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 
+[discrete]
+[[release-notes-7.17.4]]
+=== 7.17.4
+
+[discrete]
+[[bug-fixes-7.17.4]]
+==== Bug fixes and enhancements
+
+* {kibana-ref}/pre-configured-connectors.html[Preconfigured connectors] can be used with cases ({pull}130372[#130372]).
```
Comment (jmikell821) [inline docs/release-notes/7.17.asciidoc:12]:
```suggestion
* Allows {kibana-ref}/pre-configured-connectors.html[preconfigured connectors] to be used with cases ({pull}130372[#130372]).
```

**Review by jmikell821 — COMMENTED** [review]
Slight edit - then good to go. Thanks!

**Pull request merged**